.circleci/config.yml

```diff
@@ -162,7 +164,7 @@ workflows:
             tags:
               # match tags such as "2022.03.08" or "2022.03.08-1", with or
               # without a -stage suffix.
-              only: /^20\d{2}\.\d{2}\.\d{2}(?:-\d+)?(?:-stage)?$/
+              only: /^20\d{2}\.\d{2}\.\d{2}(-\d+)?(-stage)?$/
```

Comment:
these last two still have `\d` in them?

Reply:
Yes, I debated changing them... it's fine in circleci config...

Reply:
https://github.com/mozilla/extension-workshop/pull/1258/commits/7aeb2403051d367518993084bd9aac6231f8dd55